easybuild/easyconfigs/c/COMSOL/COMSOL-6.2.0.290.eb

```diff
@@ -0,0 +1,24 @@
+name = 'COMSOL'
+version = '6.2.0.290'
+
+homepage = 'https://www.comsol.se/'
+description = """
+COMSOL Multiphysics is a general-purpose software platform, based on
+advanced numerical methods, for modeling and simulating physics-based
+problems. 
+"""
+
+toolchain = SYSTEM
+
+# Note: sources from COMSOL are often named poorly. You can view the exact version in the "ver" file inside the iso:
+# 7z x COMSOL-*.iso ver; cat ver
+sources = ['COMSOL-%(version)s.iso']
+checksums = ['a90744cb29379a871ca53a0f525a0c378155f5facb3cb5e4dd540ccfc7f4eeec']
+
+download_instructions = 'Obtain from comsol.se'
+
+osdependencies = [('p7zip-plugins', 'p7zip-full'), 'csh']  # for extracting iso-files
+
+# license_file = 'license.dat'  # or EB_COMSOL_LICENSE_FILE
```

Comment:
```suggestion
# license_file = 'license.dat'  # or EB_COMSOL_LICENSE_FILE or LMCOMSOL_LICENSE_FILE or LM_LICENSE_FILE
```

Reply:
Erh.. why would we actively encourage the use of the old variable name. I think this addition is just worse.